rest/route/github.go

```diff
@@ -90,12 +90,14 @@ func (gh *githubHookApi) Parse(ctx context.Context, r *http.Request) error {
 
 	gh.event, err = github.ParseWebHook(gh.eventType, body)
 	if err != nil {
-		grip.Error(message.WrapError(err, message.Fields{
-			"source":  "github hook",
-			"msg_id":  gh.msgID,
-			"event":   gh.eventType,
-			"message": "rejecting github webhook",
-		}))
+		if gh.eventType != "pull_request_review_thread" {
```

Comment:
since we know from the beginning that we're not going to parse this event maybe we can return early on line 76? 
I could also imagine that this wouldn't be an error: we could return a 200 to tell GitHub the message was processed successfully (we've pulled everything we want from it, which in this case is nothing)

Reply:
i thought it would be more clear to wrap the validate and parse in an if statement rather than return early.

Reply:
I think you're right and from the looks of it, the gh.event being nil will ultimately give us the error we want in Run.